rfc6962-bis.xml

```diff
@@ -338,7 +338,7 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
     </section>
     <section title="Private Domain Name Labels">
       <t>
-        Some regard some DNS domain name labels within their registered domain space as private and security sensitive. Even though these domains are often only accessible within the domain owner's private network, it's common for them to be secured using publicly trusted TLS server certificates. We define a mechanism to allow these private labels to not appear in public logs.
+        Some organizations regard certain DNS domain name labels within their registered domain space as private and security sensitive. Even though these domains are often only accessible within the domain owner's private network, it's common for them to be secured using publicly trusted TLS server certificates. We define a mechanism to allow these private labels to not appear in public logs.
```

Comment:
Any of the proposed ("Some regard certain DNS names..." or "Some domain registrants regard certain DNS domain name labels within their registered domain space as private and security sensitive." ) are more than fine with me -- as is simply leaving it how it was, etc. This was just a nit.

Please let me know if you'd like a new PR or if you'd prefer I just close this (Issues in Github work nicely for substantive changes, but for nits it often just feels like busy-work).